### PR TITLE
add missing global attributes to turtle

### LINKED ISSUE #1392: turtle module missing globals from Turtle and Screen class
The turtle module is missing some global methods.

e.g.

```python
import turtle
turtle.listen()
```

This should work - but in Skulpt we haven't added the `listen()` method from the `Screen` class to the `turtle` module scope.

### PR DESCRIPTION
**Pull request by alexwenbj**
See #1393.

close #1392

### CODE REVIEW

File: src/lib/turtle.js [hunk 1]
```diff
@@ -2337,6 +2337,18 @@ function generateTurtleModule(_target) {
...
+    addModuleMethod(Screen, _module, "$listen", getScreen);
+    addModuleMethod(Screen, _module, "$register_shape", getScreen);
+    addModuleMethod(Screen, _module, "$clearscreen", getScreen);
+    addModuleMethod(Screen, _module, "$bgcolor", getScreen);
+    addModuleMethod(Screen, _module, "$bgpic", getScreen);
+    addModuleMethod(Screen, _module, "$setworldcoordinates", getScreen);
+    addModuleMethod(Screen, _module, "$ontimer", getScreen);
+    addModuleMethod(Screen, _module, "$onscreenclick", getScreen);
+    addModuleMethod(Screen, _module, "$exitonclick", getScreen);
+    addModuleMethod(Screen, _module, "$colormode", getScreen);
```
[the remark this answers is not in the document]
Comment (alexwenbj) [reply src/lib/turtle.js:2351]: The following code can be run:
```
import turtle
turtle.colormode(255)
turtle.done()
```
and in the [turtle doc](https://docs.python.org/3/library/turtle.html#turtle.colormode),it says:

[the remark this answers is not in the document]
Comment (fangjiapeng) [reply src/lib/turtle.js]: hello，I wonder how to add `$screensize` to `Turtle`'s prototype.

**Comment by s-cork** [comment]
@bnmnetp maybe you can have a look at this PR?

**Comment by alexwenbj** [comment]
Sorry it seems I deleted this PR accidently , I will re-open a PR for this.

**Comment by alexwenbj** [comment]
@s-cork @bnmnetp Please review #1445 .

**Pull request closed without merging**